lib/rules/lint-unless-helper.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const buildPlugin = require('./base');
+
+const messages = {
+  followingElseBlock: 'Invoking an `{{else}}` helper with the `unless` helper is not recommended',
+  followingElseIfBlock: 'Invoking an `{{else if}}` helper with the `unless` helper is not recommended',
+  asElseUnlessBlock: 'Invoking an `{{else unless}}` helper is not recommended',
+  withHelper: 'Invoking an `{{unless}}` helper with another template helper is not recommended'
+};
+
+function logMessage(context, message, loc, actual) {
```

Comment:
If you put this on your `class` below, you can call it via `this.logMessage()` and not pass in `context`.  I don't have a significant preference, just curious if you do 😄...